docs/architecture/application/forms.md

```diff
@@ -127,4 +127,306 @@ def initial_input_form_generator(product: UUIDstr, product_name: str) -> FormGen
     return {**user_input.dict(), **user_input_node.dict()}
 ```
 
+## custom form field
+
+You can create a custom field component in frontend and can take reference from components in `orchestrator-gui/src/lib/uniforms-surfnet/src`.
```

Comment:
You can create a custom field component in the frontend. The components in `orchestrator-gui/src/lib/uniforms-surfnet/src` can be used to study reference implementations for a couple of custom form field types.